apps/meteor/server/features/EmailInbox/EmailInbox_Incoming.ts

```diff
@@ -170,6 +170,10 @@ export async function onEmailReceived(email: ParsedMail, inbox: string, departme
 	const msgId = Random.id();
 
 	logger.debug(`Sending email message to room ${rid} for visitor ${guest._id}. Conversation assigned to department ${department}`);
+
+	if (room?.onHold && (LivechatRooms as any)?.unsetOnHold) {
```

Comment:
You can use the raw model to avoid this kind of typecasting 👀 (the one from @rocketchat/models)

Reply:
it seems that was only needed because we were not setting the token, so the message appeared like not coming from the user in some livechat message checks (like automatic off from onHold), is this version better?

Reply:
This change won't be required since the on-hold feature wasn't working properly with the email channel due to a bug. I've raised this PR to fix that issue https://github.com/RocketChat/Rocket.Chat/pull/26363

Reply:
FYI guys: I've merged https://github.com/RocketChat/Rocket.Chat/pull/26363 over here so this branch should have the onhold fix